dw-text-editor.js

```diff
@@ -387,11 +387,10 @@ class DwTextEditor extends LitElement {
    * @param {String} eventName Event Name
    * @param {Object} event Event
    */
-  __dispatchProxyEvent(event) {
-    const eventName = event.type;
-    const bubbles = event.bubbles;
-    const composed = event.composed;
-    this.dispatchEvent(new CustomEvent(eventName, { detail: {event}, bubbles, composed }));
+   __dispatchProxyEvent(e) {
+    const eventName = e.type;
+    const proxyEvent = new e.constructor(eventName, e);
```

Comment:
@nirmalbaldaniya-dream From where did you receive this hint/reference? I am not convinced that this is a proper way to build an Event instance. Because each Event class's constructor might not have 2 arguments: name & options. In addition, 2nd argument should be options, while we are sending the event instance. So, how this would work?

Reply:
I got hint from here: https://stackoverflow.com/questions/11974262/how-to-clone-or-re-dispatch-dom-events